src/tests/ftest/util/server_utils.py

```diff
@@ -373,6 +373,7 @@ def clean_files(self, verbose=True):
             verbose (bool, optional): display clean commands. Defaults to True.
         """
         clean_cmds = []
+        NUM_IO_SERVERS = 2
```

Comment:
I don't think this is a safe assumption to hard-code here. There must be some other place where it's set in order to generate the server configurations. @phender: Any advice?